android-components/components/browser/state/src/main/java/mozilla/components/browser/state/reducer/TranslationsStateReducer.kt

```diff
@@ -83,6 +83,15 @@ internal object TranslationsStateReducer {
                     }
                 }
 
+                TranslationOperation.FETCH_IS_ENGINE_SUPPORTED -> {
+                    state.copyWithTranslationsState(action.tabId) {
+                        it.copy(
+                            isEngineSupported = true,
```

Comment:
Just wondering if the fetch is successful why we should default to that the engine is supported

Reply:
Thanks, great catch! I updated to be more like `FETCH_SUPPORTED_LANGUAGES`, since we don't necessarily know it should be `true` and removed setting the engine state.

(I originally was thinking it was more like the `Translate` and `Restore` cases where we can infer the state. However, 100% spot on that success != true in this case.)